app/lib/common/widgets/spaces/space_selector_drawer.dart

```diff
@@ -16,13 +16,17 @@ Future<String?> selectSpaceDrawer({
     enableDrag: true,
     context: context,
     isDismissible: true,
-    builder: (context) => SelectRoomDrawer(
-      key: key,
-      canCheck: canCheck,
-      currentSpaceId: currentSpaceId,
-      title: title ?? Text(L10n.of(context).selectSpace),
-      keyPrefix: 'select-space',
-      roomType: RoomType.space,
+    isScrollControlled: true,
+    builder: (context) => Container(
+      constraints: BoxConstraints(maxHeight: 700),
```

Comment:
Shouldn't we use ```MediaQuery``` here to determine fixed size ? The pixels might calculate differently based on device ratio...